hathor/transaction/resources/transaction.py

```diff
@@ -195,6 +200,18 @@ def get_one_tx(self, request):
 
         return json_dumpb(data)
 
+    def _validate_index(self, request: Request) -> bytes | None:
+        """Return an error message if the tokens index is disabled, and None otherwise."""
```

Comment:
Maybe change the docstring to "Return None if validation is successful, otherwise return an error message.".

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1069/commits/a5db4fc4a5c1a0e7058df7bfd03dde3d9e3b027e

Reply:
Thanks!